src/Tribe/Views/V2/Views/By_Day_View.php

```diff
@@ -406,8 +474,20 @@ protected function backfill_multiday_event_ids( array $event_ids = [] ) {
 			return;
 		}
 
+		// Prefetch provided events in a single query.
+		$event_results = tribe_events()
+			->in( $event_ids )
+			->per_page( -1 )
+			->all();
+
+		// Massage events to be indexed by event ID.
+		$events = [];
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: ff28e7191b02f8b11226c3747238aa83</sub>